react/src/components/ListItemRouterLink.tsx

```diff
@@ -23,7 +23,7 @@ export default function ListItemRouterLink(props: ListItemRouterLinkProps) {
                     {...linkProps}
                     onMouseEnter={() => setOpenTooltip(true)}
                     onMouseLeave={() => setOpenTooltip(false)}
-                />
+                ></Link>
```

Comment:
Is this change necessary?

Reply:
I think there was a deprecation warning that requires Link to have children nodes. It's totally fine if we remove it though since we don't have any children to begin with!

Reply:
OK sounds good -- the children should be passed through the props, so hopefully it doesn't complain if you revert!